plugins/che-plugin-analytics/che-plugin-analytics-wsagent/src/main/java/com/redhat/che/plugin/analytics/wsagent/AnalyticsEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package com.redhat.che.plugin.analytics.wsagent;
+
+import static com.redhat.che.plugin.analytics.wsagent.EventProperties.*;
+
+public enum AnalyticsEvent {
+  WORKSPACE_STARTED("Start Workspace in Che", 3),
```

Comment:
Since these are events I would rather use the simple past in the event name:
```
s/Start/Started/
s/Use/Used/
s/Keep/Kept/
s/Stop/Stopped/
s/Edit/Edited/
```

Reply:
In fact in Woopra by default the event label is built the following way: `Did <event name>`

This is why I thought that having verbs at the present would be better suited, because it finally renders like this in the UI:

`Did start workspace in che`

If I use the simple past, I get

`Did started workspace in che`

which is quite ugly IMO.

And to change the way event labels are shown in Woopra, you need to change the default schema of each event which requires edit rights I don't have.

In fact here it's more the name / ID of the immediate event than it's label.

Reply:
Fixed